sdk/storage/Azure.Storage.Files.DataLake/api/Azure.Storage.Files.DataLake.netstandard2.0.cs

```diff
@@ -694,6 +711,7 @@ public partial class FileSystemItem
     public partial class FileSystemProperties
     {
         internal FileSystemProperties() { }
+        public string DefaultEncryptionScope { get { throw null; } }
```

Comment:
Should this be just `EncryptionScope` ?
I'm assuming that it's called default now because it's inherited by paths under the file system.

(Assuming that we some day migrate to blob endpoint and this is possible)
How does this feature work for paths? I.e. if I create path /a/b with encryption scope would /a/b/c inherit it or not?

Reply:
Pattern match from [BlobContainerEncryptionScopeOptions](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/storage/Azure.Storage.Blobs/src/Models/BlobContainerEncryptionScopeOptions.cs#L16).

You are correct on how the functionality works.  Note that the default encryption scope concept only applies to File Systems, not Files or Directories.